packages/design-system/src/components/label.tsx

```diff
@@ -0,0 +1,78 @@
+import type { LabelHTMLAttributes, ReactNode, Ref } from "react";
```

Comment:
Please add a header like this: https://github.com/webstudio-is/webstudio-designer/blob/78af54b0175b7f7be67f1f1c1964b1ea24e10cf6/packages/design-system/src/components/button.tsx#L1-L4

So we would know that this is not a random component but something that should match design in Figma